daemon/gamemode.c

```diff
@@ -399,6 +399,8 @@ bool game_mode_context_register(GameModeContext *self, pid_t client)
 		return false;
 	}
 	cl->executable = game_mode_context_find_exe(client);
+	if (!cl->executable)
+		return false;
```

Comment:
Should be `goto error_cleanup`, this'll leak `client`.

Reply:
Okay, I'll look into it. This is also old code where we found problems. :-)

Reply:
Scrap that... It's new code by me. But what about the other above, the OOM thing?

Reply:
Scrap that again... Good morning to me. :-)

Reply:
I see potential for cleanup here anyways, will push with the next rebase.

Reply:
I cleaned up the error path and general handling in the register routine. You may want to take a look.